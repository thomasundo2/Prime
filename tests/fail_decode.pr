int main()
{
    decode(0);
    return 0;
}