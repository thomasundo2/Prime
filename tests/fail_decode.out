Fatal error: exception Failure("illegal argument int expected lint in 0")
